### ISSUE TITLE
[Usable- PowerApps Admin- Select a template]: JAWS announces incorrect row, column count in tab mode under Select a template screen

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7075]
#### Actual behavior:
JAWS announces incorrect row, column count in tab mode. JAWS announces it as 'table  0 rows and 0 columns'. 
 
#### Expected behavior:
JAWS should announce correct row, column count in tab mode. JAWS should announce it as 'table contains 26 rows and 7 columns'.
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by aneeshack4** [comment]
@sonimishra Hi Soni, which component are you seeing this issue on? If it's GroupedList or DetailsList, @natalieethell has already fixed this in this PR https://github.com/OfficeDev/office-ui-fabric-react/pull/7063#event-1958055742 and it'll be merged soon. If it's Facepile, I'm working on a fix for it and that issue's being tracked with https://github.com/OfficeDev/office-ui-fabric-react/issues/6758. Please specify the component so we can investigate further.

**Comment by natalieethell** [comment]
Hey @sonimishra thanks for submitting this issue! I'm having trouble reproducing this behavior. I just ran JAWS on the basic DetailsList example on our demo site in Edge (https://developer.microsoft.com/en-us/fabric#/components/detailslist/basic) and when tabbing to the table, it read out 

**"Entering table with 3 columns and 201 rows
Toggle selection checkbox not checked"**

On Chrome, an overview of the DetailsList is not read out like in Edge, but it does not read out "0 rows, 0 columns" either.

What browser are you seeing this on? What does your DetailsList look like? If you could provide an isolated CodePen repro, that would help us a lot. Here's a basic DetailsList, and I don't seem to see the issue here either: https://codepen.io/naethell/pen/qQbYRM. Thanks!

**Comment by sonimishra** [comment]
Hi @natalieethell, We are following up with product team on your query, will update you once we get answers from them.

**Comment by natalieethell** [comment]
Hey @sonimishra any updates? Let us know if we can help!
